src/Symfony/Component/HttpClient/CurlHttpClient.php

```diff
@@ -307,15 +305,15 @@ private static function handlePush($parent, $pushed, array $requestHeaders, \std
         }
 
         if (!isset($headers[':method']) || !isset($headers[':scheme']) || !isset($headers[':authority']) || !isset($headers[':path']) || 'GET' !== $headers[':method'] || isset($headers['range'])) {
-            $logger && $logger->info(sprintf('Rejecting pushed response from "%s": pushed headers are invalid', $origin));
+            $logger && $logger->warning(sprintf('Rejecting pushed response from "%s": pushed headers are invalid', $origin));
```

Comment:
debug

Reply:
Why ? something abnormal happened

Reply:
not really - we don't control what the remote sends